### PR TITLE
Fix Custom Template check and Add ability to redact entire field

### PR DESCRIPTION
**Pull request by MEseifan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,6 +84,7 @@
     <junit.version>4.12</junit.version>
     <slf4j.version>1.7.5</slf4j.version>
     <spark.version>2.3.1</spark.version>
+    <google.core.version>1.93.3</google.core.version>
```
Comment (nitinmotgi) [inline pom.xml]: Will this affect when use in combination with other google-cloud connectors ?

**Comment by nitinmotgi** [comment]
Minor comment that requires attention.

**Comment by nitinmotgi** [comment]
Rest of it looks good to me.

File: pom.xml (hunk 1 above)
Comment (MEseifan) [reply pom.xml]: Oh good catch, these changes were added as part of testing. They didnt affect anything so I'll revert this file

**Pull request merged**